specification.md

```diff
@@ -344,3 +337,183 @@ In this case, if the `source` value is `source-abc` and the `booking-referral` i
+      "value": "CV1654321"
+    },
+    {
+      "system": "https://flynnspharmacy.org/",
+      "value": "123"
+    }
+  ],
+  "name": "Flynn's Pharmacy in Pittsfield, MA",
+  // additional Location fields here...
+}
+```
+
+In this example, `https://flynnspharmacy.org/` is an arbitrary string that defines “Flynn’s Pharmacy” as the identifier system. _Slot Aggregators_ should only use a URL that is not under their control in cases where the URL is predictable and might reasonably be chosen by other _Slot Aggregators_. When this is not the case, _Slot Aggregators_ SHOULD choose a URL under their control. For example, an aggregator at `usdigitalresponse.org` might choose a URL like `https://fhir.usdigitalresponse.org/identifiers/flynns`.
+
+When the source system is a SMART Scheduling Links implementation, a _Slot Aggregator_ SHOULD use [FHIR’s `Resource.meta.source` field][resource_meta] to describe it. The value is a URI that SHOULD include the URL of the source system, and MAY add the resource `id`.
+
+For example, given the above example resource at `https://api.flynnspharmacy.org/fhir/smart-scheduling/$bulk-publish`, a _Slot Aggregator_ might publish a location like:
+
+```js
+{
+  "resourceType": "Location",
+  "id": "456",
+  "meta": {
+    "source": "https://api.flynnspharmacy.org/fhir/smart-scheduling/Location/123"
+  },
+  "identifier": [
+    {
+      "system": "https://cdc.gov/vaccines/programs/vtrcks",
+      "value": "CV1654321"
+    },
+    {
+      "system": "https://flynnspharmacy.org/",
+      "value": "123"
+    }
+  ],
+  "name": "Flynn's Pharmacy in Pittsfield, MA",
+  // additional Location fields here...
+}
+```
+
+
+### Data “Freshness”
+
+Aggregators may request or receive information from publishers at different times, and understanding how recently data about a Location, Schedule, or Slot was retrieved can help end users gauge the accuracy of items in an aggregated dataset. _Slot Aggregators_ SHOULD use the `lastSourceSync` extension on the `meta` field of any resource to indicate the last time at which the data was known to be accurate:
+
+```js
+{
+  "resourceType": "Schedule",
+  "id": "123",
+  "meta": {
+    "extension": [{
+      "url": "http://hl7.org/fhir/StructureDefinition/lastSourceSync",
+      "valueDateTime": "2021-04-19T20:35:05.000Z"
+    }]
+  }
+  // additional Schedule fields here...
+}
+```
+
+**Note:** `lastSourceSync` has been tentatively approved, but is not yet finalized as part of FHIR as of April 28, 2021. (You can keep up-to-date on the status of this extension in FHIR’s issue tracker at [FHIR-31567][].)
+
+
+### Unknown Availability, Capacity, or Slot Times
+
+Because source systems may experience errors or may not conform to the SMART Scheduling Links specification, _Slot Aggregators_ need additional tools to describe unusual situations that are not relevant to first-part _Slot Publishers_. Specifically, _Slot Aggregators_ may describe locations with:
```

Comment:
```suggestion
Because source systems may experience errors or may not conform to the SMART Scheduling Links specification, _Slot Aggregators_ need additional tools to describe unusual situations that are not relevant to first-part _Slot Publishers_. Specifically, _Slot Aggregators_ may describe Schedules with:
```

Is "schedules" more accurate, or am I misunderstanding?

Reply:
Yep, that would be more accurate. I got tied up in my COVID-centric use case. 😅